helm/teleport-kube-agent/templates/delete_hook.yaml

```
@@ -84,7 +84,7 @@ spec:
                 fieldPath: metadata.namespace
           - name: RELEASE_NAME
             value: {{ .Release.Name }}
-        image: {{ include "teleport-kube-agent.image" . | quote }}
+        image: '{{ include "registry" . }}/{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}'
```

```suggestion
        image: '{{ include "registry" . }}/{{ .Values.image.repository }}:{{ include "teleport-kube-agent.version" . }}'
```

We need above changes because it breaks `.Values.teleportVersionOverride` and we rely on that to set overrides, for e.g:

```
(teleport.giantswarm.io-grizzly)> kubectl get configmap t-fwtx3euk99nmazz5yg-teleport-kube-agent-config -n org-t-ahmtwex3bc9e5rma0h -o yaml
apiVersion: v1
data:
  values: |-
    roles: "kube"
    authToken: "9b9599e2-4887-47aa-962a-c7e2851c350b"
    proxyAddr: "teleport.giantswarm.io:443"
    kubeClusterName: "grizzly-t-fwtx3euk99nmazz5yg"
    teleportVersionOverride: "15.1.7"
```